packages/app/src/app/pages/Dashboard/Content/routes/Deleted/index.tsx

```diff
@@ -70,7 +74,16 @@ export const Deleted = () => {
         showSortOptions
         templates={getPossibleTemplates(sandboxes.DELETED)}
       />
-      <VariableGrid page={pageType} items={items} />
+      {items.length > 0 ? (
+        <VariableGrid page={pageType} items={items} />
+      ) : (
+        <EmptyPage.StyledWrapper>
+          <EmptyPage.StyledDescription
```

Comment:
Is there a reason for using dangerouslySetInnerHTML? `StyledDescription` is a tag with children, so you can also render the text inside unless I'm missing something? 
`>{ DESCRIPTION }</>`

Reply:
Oh, now I see the `<br />` tag. does it work if you place two `p` tags instead of one?

Reply:
Decided to leave it as is after talking in Slack.